src/monthlychallenges/index.njk

```diff
@@ -12,6 +12,31 @@ homePageBlocks:
   collectionKey: monthlychallenges
 ---
 
+<div class="bg-light py-3">
+  <div class="container-xl">
+    <h2 class="display-5">Why should members participate in the monthly challenge?</h2>
+
+    <p>
+      These monthly challenges provide members the opportunity to learn, grow and receive support and mentorship. 
+      There will be a theme for each month's challenge and weekly goals for the members to work on. 
+      Instructions, resources and additional help for the challenges is provided in the #monthly-challenge channel in slack. 
```

Comment:
We use the Oxford comma (the comma that goes before the last item in the list).

So here, let's have:

> Instructions, resources, and additional

Reply:
Same for line 20 above.